src/lib/resources/backup/schedule/flags.ts

```diff
@@ -0,0 +1,21 @@
+import { Flags } from "@oclif/core";
+
+export const backupScheduleFlagDefinitions = {
+  description: Flags.custom<string>({
+    summary: "Set the description for the backup schedule.",
+    description:
+      "Set the description for the given backup schedule to be displayed in mStudio and with the list command.",
+  }),
+  schedule: Flags.custom<string>({
+    summary: "Set the interval at which the backup should be scheduled.",
+    description:
+      "Must be specified as a cron schedule expression. " +
+      "Cannot be scheduled more often than once per hour. " +
+      "Defines the interval at which the backup creation will be executed.",
+  }),
+  ttl: Flags.custom<string>({
+    summary: "Define the backup retention period in days for backups created.",
+    description:
+      "Must be specified as a natural number between 7 and 400, representing the number of days the backup will be kept.",
+  }),
```

Comment:
Refresh my memory: Is the format like `7d` required? If so, this should be mentioned in the description. If not, this should better be of type `number`.

Reply:
format `[amount]d` is correct. will adapt the description

Reply:
come to think of it though the cli could take the guesswork off the user’s hands for whom the input as a pure number might be more convinient. But that will add some magic to the cli wich might be contradictionary to when talking directly to the api.